src/wp-includes/class-wp-scripts.php

```diff
@@ -390,8 +430,20 @@ public function do_item( $handle, $group = false ) {
 			return true;
 		}
 
+		if ( '' !== $strategy ) {
+			$strategy = ' ' . $strategy;
+			if ( ! empty( $after_non_standalone_handle ) ) {
+				$strategy .= sprintf( " onload='wpLoadAfterScripts(\"%s\")'", esc_attr( $handle ) );
+			}
+		}
 		$tag  = $translations . $cond_before . $before_handle;
-		$tag .= sprintf( "<script%s src='%s' id='%s-js'></script>\n", $this->type_attr, $src, esc_attr( $handle ) );
+		$tag .= sprintf(
+			"<script%s src='%s' id='%s-js'%s></script>\n",
+			$this->type_attr,
```

Comment:
ditto, can we really trust `$this->type_attr`?

Reply:
@adamsilverstein yes, this is not an addition by us, this is a private var in the `WP_Scripts` class, and is actually only ever assigned a value in the `init()` method of said class, where it is given a value of ` type='text/javascript'` if the current theme supports `html5` and `script`.

Please see [here](https://github.com/WordPress/wordpress-develop/blob/6.2/src/wp-includes/class-wp-scripts.php#L157).